docs/usage/runtime.md

```diff
@@ -169,7 +179,9 @@ the `runtime` transformer however turns this into:
 ```javascript
 "use strict";
 
-var _babelHelpers = require("babel-runtime/helpers");
+var _interopRequireDefault = require("babel-runtime/helpers/interop-require-default").default;
```

Comment:
This should be the output, yours might be reading from a babelrc or something, but the [es3.memberExpressionLiterals transformer](http://babeljs.io/docs/advanced/transformers/es3/member-expression-literals/) should be transforming `.default` to `['default']`.
```sh
➜  babel.github.io git:(master) echo "import foo from 'bar';" | babel --optional runtime
'use strict';

var _interopRequireDefault = require('babel-runtime/helpers/interop-require-default')['default'];

var _bar = require('bar');

var _bar2 = _interopRequireDefault(_bar);
```